excise_management/data/ir_model_fields.xml

```diff
@@ -257,4 +257,80 @@ for template in self:
         <field name="relation">uom.uom</field>
         <field name="readonly" eval="True"/>
     </record>
+    <record id="account_tax_deposit_field" model="ir.model.fields">
+        <field name="name">x_is_deposit</field>
+        <field name="field_description">Is Deposit</field>
```

Comment:
There is already a similar field on account.tax.group. But it is hidden. Maybe we should just display this one instead.